task3/whats_my_ip.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+local_ip=$(ifconfig | grep -o '192\.168\.[0-9]\+\.[0-9]\+' | head -1)
+
+  echo "$local_ip"
```

Comment:
Don't put inverted commas around the variables, Just the dollar sign will do the work.